.githooks/pre-commit

```diff
@@ -57,4 +57,6 @@ done
 # If there are whitespace errors, print the offending file names and fail.
 git diff-index --check --cached $against --
 
+npm shrinkwrap --dev
```

Comment:
You should update the deps manually. Not as part of the commit hook. Otherwise it can lead to surprises and/or insanity.